### ISSUE TITLE
When moving the first drawn polygon, the shades do not update/refresh

### ISSUE THREAD

**Issue by basslet** [fhk/50shades#1]
<img width="1488" alt="image" src="https://github.com/fhk/50shades/assets/19800246/6d6b8ab2-eb90-4357-a423-5dac991056e0">

Steps: 

- open https://fhk.github.io/50shades/
- draw polygon
- wait for shades to be drawn
- move polygon

Results: 
shades don't update

Expected: 
shades get redrawn

**Comment by fhk** [comment]
Yes the current behavior is single submission submit when edit finished.

I can change it by adding a "submit" button and overwrite the result each time.
